### PR TITLE
😁(main): add default remote

### PR DESCRIPTION
**Pull request by tuyunlei**

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -20,20 +21,28 @@ fn main() -> anyhow::Result<()> {
...
     let branch_names = repo.branches(None)?
         .try_filter_map(|(branch, _)| {
             // get name when not Error
             branch.name().map(|name| {
                 // get name when not None
                 name.map(|name| {
                     if branch.get().is_remote() {
-                        name.strip_prefix(remote_name)
-                            .map(|name| name.to_string())   // get own string when stripped
+                        name.strip_prefix(&remote_name_prefix)
```
Comment (W-Mai) [inline src/main.rs:44]: &String can be automated transformed to be a &str?

File: src/main.rs [hunk 2]
```diff
@@ -20,20 +21,28 @@ fn main() -> anyhow::Result<()> {
...
     let repo_path = args.repo.as_deref().unwrap_or(".");
     let repo = open_repo(repo_path)?;
 
+    // Obtain remote name from args or use default
+    let remote_name = if let Some(name) = args.remote { name } else {
+        let remotes = repo.remotes()?;
+        if remotes.len() != 1 {
+            return Err(anyhow!(AppError::RemoteNotSpecified));
+        }
+        remotes.get(0).unwrap().to_string()
```
Comment (W-Mai) [inline src/main.rs:33]: is 0 the original branch?

File: src/main.rs (hunk 1 above)
Comment (tuyunlei) [reply src/main.rs:44]: yes. it called [defref coercions](https://doc.rust-lang.org/book/ch15-02-deref.html?#implicit-deref-coercions-with-functions-and-methods)

File: src/main.rs (hunk 2 above)
Comment (tuyunlei) [reply src/main.rs:33]: i have checked remotes' length, so it's definitly the only remote.

**Pull request merged**